Corona-Warn-App/src/main/res/values-de/strings.xml

```diff
@@ -914,6 +914,18 @@
     <!-- XACT: other warning - illustration description, explanation image -->
     <string name="submission_positive_other_illustration_description">"Ein Smartphone übermittelt einen positiven Testbefund verschlüsselt ins System."</string>
 
+    <!-- Submission European Federal Gateway Server Consent -->
+    <!-- XHED: Page title for the positive result European Federal Gateway Server Consent page -->
+    <string name="submission_european_federal_gateway_server_consent_title">"Title Dummy Text"</string>
```

Comment:
If a text isn't defined please add `Lorem Ipsum` so that everyone sees that this is just a placeholder, you could refer to our `lorem_ipsum`.

Reply:
that's also relevant for the others

Reply:
Ok, I'll will do this.